### DISCUSSION TITLE
General question about "mass transfers"

### DISCUSSION THREAD

**Discussion by lthibeault** [smartcontractkit/full-blockchain-solidity-course-py#104] [Q&A] [answered]
Do you know of a way to send tokens from a contract to a list of all addresses (an array or mapping even) without using a loop(as I know gas fees for this can become a huge issue.
I have a contract, that when someone new enters it, I want to send a specific amount of a token to all the other addresses that are currently "participating". I cannot think of a way of either setting the "balance" manually, or paying out automatically, that does not require looping through a list.

Any assistance is greatly appreciated.

**Comment by lthibeault** [comment]
I am seeing "Iterable Mapping" as a possible solution to this. and I may not be using transfers, but just updating balances as "needed"
If you know of a better way of doing this, it would be greatly appreciated.

**Comment by PatrickAlphaC** [comment, marked as answer]
Great question. Can you actually ask this as a stack exchange Ethereum question? And leave this as a channel to ask questions about the course? 

I love how you're asking it though!

https://ethereum.stackexchange.com/

**Reply by lthibeault** [reply]
I can, and will, thank you, honestly didn't really even think about stack exchange.!
